src/trousse/feature_fix.py

```diff
@@ -353,7 +343,7 @@ def encode_single_categorical_column(
     dataset_encoded.track_history(
         FeatureOperation(
```

Comment:
You did not modify these lines in this branch but... Can we really instantiate a FeatureOperation with these arguments?
By the way, in the future, the `operation_type` should be removed because it is implicitly defined by the name of the specific FeatureOperation subclass